components/component.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/components/avatar.css">
+    <link rel="stylesheet" href="/components/component.css">
+    <link rel="stylesheet" href="/components/alert.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Documentation</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="nav-bar flex">
+            <h1>Get Started</h1>
+            <div class="horizontal-center">
+                <a class="round-border" href="/index.html">Home</a>
+            <a class="round-border" href="/components/component.html">Docs</a>
+            </div>
+            
+        </nav>
+        <main class="content-box">
+            <div class="side-container flex-column"> <h2 class="color-white margin-bottom-one text-center rounded-square-box">Components</h2>
+                <div class="sidebar-menu">
+                    <a href="#avatar" class="line-space sidebar-items">Avatar</a>
+                    <a href="#alert" class="line-space sidebar-items">Alert</a>
+                    <a href="#" class="line-space sidebar-items">Badge</a>
+                    <a href="#" class="line-space sidebar-items">Button</a>
+                    <a href="#" class="line-space sidebar-items">Card</a>
+                    <a href="#" class="line-space sidebar-items">Image</a>
+                    <a href="#" class="line-space sidebar-items">Input</a>
+                    <a href="#" class="line-space sidebar-items line-space-one">Text-utillities</a>
+                    <a href="#" class="line-space sidebar-items">List</a>
+                    <a href="#" class="line-space sidebar-items">Navigation</a>
+                </div>
+            </div>
+            <div class="main-container">
+                <div id="avatar" class="avatar-container margin-bottom-three">
```

Comment:
Use semantic HTML tag here. You can use `section` here instead of div. Wrap your components in section.